pandas_vs_excel/pandas_vs_excel.ipynb

```diff
@@ -0,0 +1,640 @@
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>Steve Hayes</td>\n",
+       "      <td>69</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>Cecilia Richards</td>\n",
+       "      <td>91</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>Dorothy Harris</td>\n",
+       "      <td>53</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>Jordan Davis</td>\n",
+       "      <td>76</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "       Student Name  Exam 1 Score\n",
+       "0      Edwin Duncan            83\n",
+       "1       Steve Hayes            69\n",
+       "2  Cecilia Richards            91\n",
+       "3    Dorothy Harris            53\n",
+       "4      Jordan Davis            76"
+      ]
+     },
+     "execution_count": 79,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# import pandas library\n",
+    "import pandas as pd\n",
+    "\n",
+    "# create pandas DataFrame from CSV\n",
+    "exam_one = pd.read_csv('csv/exam_one.csv', delimiter=',')\n",
+    "exam_two = pd.read_csv('csv/exam_two.csv', delimiter=',')\n",
+    "sections = pd.read_csv('csv/sections.csv', delimiter=',')\n",
+    "\n",
+    "# print the top five entires of the DataFrame\n",
+    "exam_one.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "Columns can be indiviudally selected simply by referring to them by name."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 80,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0        Edwin Duncan\n",
+       "1         Steve Hayes\n",
+       "2    Cecilia Richards\n",
+       "3      Dorothy Harris\n",
+       "4        Jordan Davis\n",
+       "Name: Student Name, dtype: object"
+      ]
+     },
+     "execution_count": 80,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "exam_one['Student Name']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "## Vlookup\n",
+    "\n",
+    "Experienced Excel users rely on Vlookup, a built-in function that searches (looks up) a specified value in one column and returns the corresponding value of another column. For our example of exam scores, we would like to take a student's second exam score and include it into the table of first exam score. The column of student names may not be in the same order, e.g., the first name in one table may not correpsond to the first name in another table.\n",
```

Comment:
This is just me being picky (and feel free to ignore this suggestion), but I would reword this sentence as, "Let us create a table that displays students' scores for the first two exams."

Reply:
your approach is clearer